src/partition.jl

```diff
@@ -281,10 +281,19 @@ graph from the results of `partition`.
 A graph partitioning backend such as Metis or KaHyPar needs to be installed for this function to work
 if the subgraph vertices aren't specified explicitly.
 """
-function partition(
-  g::AbstractGraph; npartitions=nothing, nvertices_per_partition=nothing, kwargs...
-)
-  return partition(g, subgraph_vertices(g; npartitions, nvertices_per_partition, kwargs...))
+function partition(g::AbstractGraph; npartitions = nothing, nvertices_per_partition = nothing, sgraph_vertices = nothing, kwargs...)
+
+  if(count([npartitions != nothing, nvertices_per_partition != nothing, sgraph_vertices != nothing ]) >= 2)
+    throw(error("ERROR: CANNOT SPECIFY MULTIPLY PARTITIONING OPTIONS, PLEASE GIVE ONLY 1"))
```

Comment:
In general we should not use all upper case, just use casing of normal writing (including in comments).

Reply:
Also, `error` already throws so you shouldn't need `throw`.

Reply:
Sure, I will change that.